### DISCUSSION TITLE
Citeproc: prevent cited item from appearing in bibliography

### DISCUSSION THREAD

**Discussion by p3palazzo** [jgm/pandoc#9706] [Q&A]
I am wondering if there is a Citeproc command that prevents an item from showing up in the bibliography. My use case is that I am citing several volumes of a series and only want to output the reference to the series as a whole in the bibliography (which I do via the `nocite` command), not each volume as a separate entry.

Now, why don't I cite the series and insert the volume number manually alsongside the page number, you might ask. Well, different citation styles have different ways of formatting volume numbers, so I would have to edit each citation manually when I change citation styles.

**Comment by jgm** [comment]
I don't think there is a way to do this...

**Reply by jgm** [reply]
You could use a Lua filter, applied AFTER citeproc, that just removes the item with that identifier from the formatted bibliography.

**Reply by reagle** [reply]
I use a preprocessor where I can hide citations that begin with `#`. For example, `[#@Smith2019tdv]`. I hope one day see commented citations in addition to [locator only ones](https://github.com/jgm/pandoc/issues/7205#issuecomment-851026018).
